components/com_content/src/Model/ArticlesModel.php

```diff
@@ -491,6 +491,23 @@ protected function getListQuery()
             $query->where($authorWhere . $authorAliasWhere);
         }
 
+        // Filter by checked_out status
+        $checkedOut = $this->getState('filter.checked_out', null);
+        if ($checkedOut !== null && $checkedOut !== '' && $checkedOut !== false) {
```

Comment:
```suggestion
        $checkedOut = $this->getState('filter.checked_out', null);

        if (is_numeric($checkedOut)) {
```
1. Leave an empty line before the `if` for better readability.
2. Wouldn't an `is_numeric($checkedOut)` work here, too? That's less complicated than checking for null and empty string and false.

Reply:
P.S. to 2. ... and it's used elsewhere in the same method for the same purpose.

Reply:
Thank you for your review and suggestion Sir, @richard67! 
You're right Sir—using `is_numeric($checkedOut)` would simplify the condition and improve readability.
I'll update the code accordingly and add the empty line for clarity as well.
Appreciate your attention to detail!